WcaOnRails/app/controllers/admin/avatars_controller.rb

```diff
@@ -20,7 +20,7 @@ def update_all
           when "defer"
             # do nothing!
           else
-            raise "Unrecognized avatar action '#{args[:action]}'"
+            fail "Unrecognized avatar action '#{args[:action]}'"
```

Comment:
Rubocop suggests different semantics for these methods: http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/SignalException

This make some sense, but if everyone thinks `raise == fail` [1] then it's useless and we should just stick with `raise` everywhere.

[1] To be fair: I once knew about this Rubocop rule, but had to look it up again...